scalableminds/webknossos
# Fix oblique bug II

I adapted two things:
- the amount of scan lines is determined dynamically instead of using a constant (yay, one constant less :tada:) <-- this fixed one error class
- Instead of only casting the scan lines before and after the current z (this is done to ensure buckets are immediately available when moving across bucket boundaries), this is also done at the current z <-- this solved a second error class

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- I set up an affected dataset locally and compared the before VS after scenarios to check that the bug was fixed

### Issues:
- fixes #5288 

------
